service/history/workflow/workflow_task_state_machine.go

```diff
@@ -482,23 +482,10 @@ func (m *workflowTaskStateMachine) skipWorkflowTaskCompletedEvent(workflowTaskTy
 	return onlyUpdateRejectionMessages
 }
 func (m *workflowTaskStateMachine) AddWorkflowTaskCompletedEvent(
-	scheduledEventID int64,
-	startedEventID int64,
+	workflowTask *WorkflowTaskInfo,
```

Comment:
This file is an actual change. All the rest is just update callers and tests.

Everywhere where these 3 methods are called workflow task object already exist and populated with correct data. I am now passing it directly instead of calling `m.GetWorkflowTaskInfo(scheduledEventID)` one more time.